packages/spacecat-shared-data-access/docs/schema.json

```diff
@@ -525,6 +525,64 @@
           }
         }
       }
+    },
+    {
+      "TableName": "spacecat-services-configurations",
+      "KeyAttributes": {
+        "PartitionKey": {
+          "AttributeName": "GSI1PK",
```

Comment:
Should not be called `GSI1PK` as this would refer to a Global Secondary Index attribute in a main table used in a GSI. Call it `PK` for example.